fastlane/Fastfile

```diff
@@ -355,39 +355,27 @@ platform :ios do
     update_identifiers
     replace_assets
 
-    sh 'rm -rf ../build-ios/'
-    sh 'cd ../ios/ && xcodebuild -workspace Mattermost.xcworkspace/ -scheme Mattermost -sdk iphoneos -configuration Release -parallelizeTargets -resultBundlePath ../build-ios/result -derivedDataPath ../build-ios/ CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO'
-    sh 'cd ../build-ios/ && mkdir -p Payload && cp -R ../build-ios/Build/Products/Release-iphoneos/Mattermost.app Payload/ && zip -r Mattermost-unsigned.ipa Payload/'
-    sh 'mv ../build-ios/Mattermost-unsigned.ipa ../'
-    sh 'rm -rf ../build-ios/'
+    build_ios_unsigned_or_simulator(
+      more_xc_args: "-sdk iphoneos CODE_SIGNING_ALLOWED=NO",
+      rel_build_dir: "Release-iphoneos",
+      output_file: "Mattermost-unsigned.ipa",
+    )
```

Comment:
@sadohert I made changes to iOS unsigned build. Could you please check if it works for you? Thanks!